app/src/main/java/com/android/sample/ui/authentication/Login.kt

```diff
@@ -107,15 +94,7 @@ fun LoginScreen(navigationActions: NavigationActions) {
               // Greeting text
               Text(
                   text = "Hello again,\nGood to see you back !",
-                  style =
-                      TextStyle(
-                          fontSize = 32.sp,
-                          lineHeight = 40.sp,
-                          fontFamily = FontFamily(Font(R.font.roboto)),
-                          fontWeight = FontWeight(500),
-                          color = PrimaryPurple,
-                          textAlign = TextAlign.Center,
-                          letterSpacing = 0.32.sp),
+                  style = TypographyBeatLink.displayLarge,
```

Comment:
Consider utilizing MaterialTheme.typography to further centralize theme control and maintain consistency throughout the application.

```suggestion
                  style = MaterialTheme.typography.displayLarge,
```